graphics/macgui/mactext.cpp

```diff
@@ -838,8 +857,7 @@ void MacText::resize(int w, int h) {
 }
 
 void MacText::appendText(const Common::U32String &str, int fontId, int fontSize, int fontSlant, bool skipAdd) {
-	appendTextDefault(str, skipAdd);
-
+//	appendTextDefault(str, skipAdd);
```

Comment:
If it is not needed, perhaps you could clean it up and just remove, and remove the `skipAdd` parameter?